Compilers.sln

```diff
@@ -162,26 +160,46 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Net.Compilers.Too
 EndProject
 Project("{778DAE3C-4631-46EA-AA77-85C1314464D9}") = "vbi", "src\Interactive\vbi\vbi.vbproj", "{706CFC25-B6E0-4DAA-BCC4-F6FAAFEEDF87}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.CodeAnalysis.CodeStyle.UnitTestUtilities", "src\CodeStyle\Core\Tests\Microsoft.CodeAnalysis.CodeStyle.UnitTestUtilities.csproj", "{21A59DB8-E9E0-42EB-A599-457C5BC87FC3}"
```

Comment:
Renaming `Microsoft.CodeAnalysis.CodeStyle.UnitTests` to `Microsoft.CodeAnalysis.CodeStyle.UnitTestUtilities` is the core change to the sln file and the only compiler layer change. Other changes got pulled in by VS automatically.